OWNERS_ALIASES

```diff
@@ -56,6 +56,8 @@ aliases:
   - lance
   - salaboy
   knative-admin:
+  - Vishal-Chdhry
+  - craydr
```

Comment:
```suggestion
  - creydr
```
will address it with a PR in knative/community. Maybe that's the reason for the invalid-owners-file label